### DISCUSSION TITLE
V7.0 Beta30 - Change to `|>` / `<|` ligatures

### DISCUSSION THREAD

**Discussion by matthew-e-brown** [subframe7536/maple-font#279] [General]
V7.0 Beta 30 changed the `|>` and `<|` ligatures to be a little bit longer. I use the `|>` operator a lot when writing R code, and I quite liked that the `|>` glyph was a perfect (equilateral) triangle. I am sure the new ones will grow on me in due time, so it's not a huge deal; I just don't see an open or closed issue explaining the change, so I am just curious as to why it was done. 🙂

I suppose that making the glyph longer does make it closer to being the full width of the two underlying characters. Was that why it was done?

**Comment by subframe7536** [comment]
Thanks for reporting, the gap should be visually balanced. I just realized this issue after releasing beta30. All the ligatures will be checked and fixed in next release.

**Reply by matthew-e-brown** [reply]
Oh? Is there an error with the new glyphs? I didn't notice any problem, I was mostly just curious. :)

**Reply by subframe7536** [reply]
Some unusual ligatures like reversed arrows have some rooms to improve, and their gap ratio are different between different masters and styles. Although this may not be very noticable, but it does exist.

**Reply by matthew-e-brown** [reply]
"Gap ratio" meaning?

**Reply by subframe7536** [reply]
The spacing ratio of the same character with different font weights to the contour edge on both sides.
